redirects.yaml

```diff
@@ -93,6 +93,7 @@ tv/show/13: https://youtu.be/NPLvHosF_Dg
 tv/show/14: https://youtu.be/BGGjFE_N2Yg
 tv/show/15: https://youtu.be/ColU1MSzCEI
 tv/show/16: https://youtu.be/f_BfB9Iynco
+tv/show/17: https://youtube.com/shorts/gQ99wO7_Znc
```

Comment:
Show? Soll das nicht ehr in die Short?

Reply:
ich korriegiere es

Reply:
Montag Janosch, kannst du das hier fixen oder muß ich einen neuen PR machen?

Reply:
ist korriegiert https://github.com/contao/to.contao.org/commit/5d84ea36c5d1549d0d6e1d3bcee25633054f661d

Reply:
AWESOME! Thank you